src/controllers/auth.controller.js

```diff
@@ -0,0 +1,107 @@
+import { ApiError } from '../exeptions/api.error.js';
+import { User } from '../moduls/User.js';
+import { userService } from '../service/user.service.js';
+import bcrypt from 'bcrypt';
+import { Console } from '../untils/Console.js';
```

Comment:
There is a typo in the import path: 'untils' should be 'utils'. Please correct the path to '../utils/Console.js'.